tests/Manipulators/SizeTest.php

```diff
@@ -126,6 +126,16 @@ public function testResolveMissingDimensions()
         $this->assertSame([200, 100], $this->manipulator->resolveMissingDimensions($image, null, 100));
     }
 
+    public function testResolveMissingDimensionsWithOddDimensions()
+    {
+        $image = Mockery::mock('Intervention\Image\Image', function ($mock) {
+            $mock->shouldReceive('width')->andReturn(1024);
+            $mock->shouldReceive('height')->andReturn(553);
+        });
+
+        $this->assertSame([411, 222], $this->manipulator->resolveMissingDimensions($image, 411, null));
+    }
```

Comment:
@sebastianpopp thx the fix. Might it be a good idea to also add tests for a resize with a null width?

Reply:
There's already a test for a resize with a null width above. I just wanted to add an additional test case that fails with the old implementation and passes with my fix to demonstrate why it's needed.